docs/dev/table/python/vectorized_python_udfs.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Vectorized User-defined Functions"
+nav-parent_id: python_tableapi
+nav-pos: 30
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+Vectorized user-defined functions are functions which are executed by transferring a batch of elements between JVM and Python VM in Arrow columnar format.
```

Comment:
Also add some descriptions about the advantages of the vectorized udfs over regular udfs?